From aad7c533f3d34a1ce6a4c65c4631077e1ee3502f Mon Sep 17 00:00:00 2001
From: gfxVPLsdm <gfxvplsdm@intel.com>
Date: Fri, 8 Apr 2022 20:31:35 +0800
Subject: [PATCH] [HEVCe/lib] Fix REXT DDIID issue (#2566)

Fixed REXT DDIID issue by adding missing reference symbol of PickDDIIDREXT function interface

Upstream-Status: Backport

Co-authored-by: Lightny Gong <lightny.gong@intel.com>
Signed-off-by: Teng, Jin Chung <jin.chung.teng@intel.com>
---
 _studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp b/_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp
index 1d4c7ce..eb3ae50 100644
--- a/_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp
+++ b/_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp
@@ -108,7 +108,7 @@ mfxStatus PickDDIIDNormal(VAID*& DDIID, Defaults::Param defPar, const mfxVideoPa
     return MFX_ERR_NONE;
 }
 
-mfxStatus PickDDIIDREXT(VAID* DDIID, const mfxVideoParam& par)
+mfxStatus PickDDIIDREXT(VAID* &DDIID, const mfxVideoParam& par)
 {
     auto& fi = par.mfx.FrameInfo;
     const mfxExtCodingOption3* pCO3 = ExtBuffer::Get(par);
-- 
2.7.4

